CONTRIBUTING.md

```diff
@@ -64,6 +65,23 @@ Following these will ensure that your pull request is going to be accepted.
    This might seem pedantic but we believe that in essence this is going to improve overall comment and documentation quality.
 1. If possible try to sign your commits, e.g. using GPG keys. For more information about this go [here](https://help.github.com/en/articles/signing-commits).
 
+### Backwards Compatibility
+
+ink! and pallet_contracts are the projects under active development. As contracts API functions evolve to their newer versions, we need to introduce them in ink! in a way that still keeps current *MAJOR* ink! version compatible with elder substrate nodes built with the pallet_contracts version having not this new functionality.
+
+In order to achieve this, please stick to the following workflow.
+
+Imagine there is a `[seal0] function()` in pallet\_contracts API and our `ink_env::function()` uses its import under the hood.
```

Comment:
```suggestion
Imagine there is a `[seal0] function()` in the Contract pallet API and our `ink_env::function()` imports it under the hood.
```

Reply:
well function does not import, it uses an imported one